index.html

```diff
@@ -1,1688 +1,2240 @@
+            <li>Return [=this=].{{Gamepad/[[timestamp]]}}.
+            </li>
+          </ol>
+        </dd>
+        <dt>
+          <dfn>mapping</dfn> attribute
+        </dt>
+        <dd>
+          <p>
+            The mapping in use for this device. If the user agent has knowledge
+            of the layout of the device, then it SHOULD indicate that a mapping
+            is in use by setting {{Gamepad/mapping}} to the corresponding
+            {{GamepadMappingType}} value.
+          </p>
+          <p>
+            To <dfn data-lt="selecting a mapping">select a mapping</dfn> for a
+            gamepad device, run the following steps:
+          </p>
+          <ol>
+            <li>If the button and axis layout of the gamepad device corresponds
+            with the [=Standard Gamepad=] layout, then return
+            {{GamepadMappingType/"standard"}}.
+            </li>
+            <li>Return {{GamepadMappingType/""}}.
+            </li>
+          </ol>
+        </dd>
+        <dt>
+          <dfn>axes</dfn> attribute
+        </dt>
+        <dd>
+          <p>
+            Array of values for all axes of the gamepad. All axis values MUST
+            be linearly normalized to the range [-1.0 .. 1.0]. If the
+            controller is perpendicular to the ground with the directional
+            stick pointing up, -1.0 SHOULD correspond to "forward" or "left",
+            and 1.0 SHOULD correspond to "backward" or "right". Axes that are
+            drawn from a 2D input device SHOULD appear next to each other in
+            the axes array, X then Y. It is RECOMMENDED that axes appear in
+            decreasing order of importance, such that element 0 and 1 typically
+            represent the X and Y axis of a directional stick. The same object
+            MUST be returned until the [=user agent=] needs to return different
+            values (or values in a different order).
+          </p>
+          <p>
+            The {{Gamepad/axes}} getter steps are:
+          </p>
+          <ol>
+            <li>Return [=this=].{{Gamepad/[[axes]]}}.
+            </li>
+          </ol>
+        </dd>
+        <dt>
+          <dfn>buttons</dfn> attribute
+        </dt>
+        <dd>
+          <p>
+            Array of button states for all buttons of the gamepad. It is
+            RECOMMENDED that buttons appear in decreasing importance such that
+            the primary button, secondary button, tertiary button, and so on
+            appear as elements 0, 1, 2, ... in the buttons array. The same
+            object MUST be returned until the [=user agent=] needs to return
+            different values (or values in a different order).
+          </p>
+          <p>
+            The {{Gamepad/buttons}} getter steps are:
+          </p>
+          <ol>
+            <li>Return [=this=].{{Gamepad/[[buttons]]}}.
+            </li>
+          </ol>
+        </dd>
+        <dt>
+          <dfn>vibrationActuator</dfn> attribute

```

Comment:
We should define algorithmic steps for the vibrationActuator attribute getter like we do for other attributes.

```
<p>
  The {{Gamepad/vibrationActuator}} getter steps are:
</p>
<ol>
  <li>Return [=this=].{{Gamepad/[[vibrationActuator]]}}.
  </li>
</ol>
```

I think the paragraph below is no longer needed since we now define steps for initializing the internal slot in [=initialize the vibration actuator=].